src/components/SkillPage/SkillListing.js

```diff
@@ -269,7 +269,9 @@ class SkillListing extends Component {
   };
 
   updateData = skillData => {
-    this.saveSkillRatings(skillData.skill_rating.stars);
+    if (skillData.skill_rating) {
+      this.saveSkillRatings(skillData.skill_rating.stars);
```

Comment:
This has been taken care of in https://github.com/fossasia/susi_skill_cms/pull/762. I think you can remove the changes in this section of the code. Thanks :-)